config/settings.js

```diff
@@ -27,11 +27,20 @@
  */
 
 window.spinnakerSettings = {
+  defaultProviders: ['aws', 'gce', 'azure', 'cf', 'kubernetes'],
+  feedbackUrl: process.env.FEEDBACK_URL,
   gateUrl: gateUrl,
-  bakeryDetailUrl: bakeryBaseUrl + '/api/v1/global/logs/{{context.status.id}}?html=true',
+  bakeryDetailUrl: bakeryBaseUrl + '/api/v1/global/logs/{{context.status.id}}?html=true'
+  authEndpoint: process.env.AUTH_ENDPOINT,
   pollSchedule: 30000,
-  defaultTimeZone: 'America/New_York', // see http://momentjs.com/timezone/docs/#/data-utilities/
+  defaultTimeZone: process.env.TIMEZONE || 'America/Los_Angeles', // see http://momentjs.com/timezone/docs/#/data-utilities/
   providers: {
+    azure: {
+      defaults: {
+        account: 'azure-test',
```

Comment:
I dont think you need to change anything to add new providers and have them
get picked up by the centralized config stuff. Ping me if it doesnt seem to
"just work".

On Mon, Jan 25, 2016 at 4:09 PM, Matt Duftler <notifications@github.com>
wrote:

> In config/settings.js
> <https://github.com/spinnaker/spinnaker/pull/702#discussion_r50756394>:
>
> >    providers: {
> > +    azure: {
> > +      defaults: {
> > +        account: 'azure-test',
>
> @ewiseblatt <https://github.com/ewiseblatt> Can you please point @AMeng
> <https://github.com/AMeng> in the right direction as to what (if
> anything) would need to be modified in the checked-in 'centralized config'
> stuff to enable the additional accounts to be rewritten here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spinnaker/spinnaker/pull/702/files#r50756394>.
>